addons/nametags/XEH_preInit.sqf

```diff
@@ -4,4 +4,31 @@ ADDON = false;
 
 #include "XEH_PREP.hpp"
 
+// settings
+[QUOTE(ADDON), QGVAR(showNames), "LIST", [localize CSTRING(showNames), localize CSTRING(showNames_Desc)], [
+    [0,1,2,3,4],
+    [localize ECSTRING(common,Disabled), localize ECSTRING(common,Enabled), localize CSTRING(OnlyCursor), localize CSTRING(OnlyKeypress), localize CSTRING(OnlyCursorAndKeypress)],
+    1
+], nil, FUNC(updateSettings)] call CBA_settings_fnc_create;
```

Comment:
Not much difference, I don't think anyone reads settings config in-game. Does this also make it faster?